### PR TITLE
Add guestbook sample for Java 11

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

**Comment by averikitsch** [comment]
ping @lesv, ready for a review

File: appengine-java11/guestbook-cloud-firestore/src/main/java/com/example/guestbook/Persistence.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+public class Persistence {
+
+  private static AtomicReference<Firestore> firestore = new AtomicReference<>();
+
+  @SuppressWarnings("JavadocMethod")
+  public static Firestore getFirestore() {
+    if (firestore.get() == null) {
+      // Authorized Firestore service
+      firestore.set(
+          FirestoreOptions.newBuilder().setProjectId("your-project-id").build().getService());
```
Comment (lesv) [inline appengine-java11/guestbook-cloud-firestore/src/main/java/com/example/guestbook/Persistence.java]: YOUR-PROJECT-ID perhaps?

File: appengine-java11/guestbook-cloud-firestore/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lesv) [inline appengine-java11/guestbook-cloud-firestore/src/main/webapp/WEB-INF/web.xml]: Copyright & License ?

File: appengine-java11/pom.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lesv) [inline appengine-java11/pom.xml]: Is this still relevant?

File: appengine-java11/guestbook-cloud-firestore/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```
Comment (lesv) [inline appengine-java11/guestbook-cloud-firestore/src/main/webapp/index.jsp]: Copyright & License ?

File: appengine-java11/guestbook-cloud-firestore/src/main/appengine/app.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+runtime: java11
+instance_class: F2
```
[the remark this answers is not in the document]
Comment (averikitsch) [reply appengine-java11/guestbook-cloud-firestore/src/main/appengine/app.yaml]: As of right now.

**Pull request merged**